src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -356,6 +356,70 @@ public void testCategoryIndexPageTableInSyncWithAllChecksPageTable() throws Exce
         }
     }
 
+    @Test
+    public void testAlphabetOrderInNames() throws Exception {
+        final String input = Files.readString(SITE_PATH);
+        final Document document = XmlUtil.getRawXml(SITE_PATH.toString(), input, input);
+        final NodeList sources = document.getElementsByTagName("item");
+
+        for (int position = 0; position < sources.getLength(); position++) {
+            final Node current = sources.item(position);
+
+            if ("Checks".equals(XmlUtil.getNameAttributeOfNode(current))) {
+                final List<String> groupNamesOrder = getCurrentNamesOrder(current);
+                final List<String> sortedGroupNames = new ArrayList<>(groupNamesOrder);
+                Collections.sort(sortedGroupNames);
+                assertWithMessage("Group names must be in alphabetical order.")
+                        .that(groupNamesOrder)
+                        .containsExactlyElementsIn(sortedGroupNames)
+                        .inOrder();
+
+                Node groupNode = current.getFirstChild();
+                int index = 0;
+                final int totalGroups = XmlUtil.getChildrenElements(current).size();
+                while (index < totalGroups) {
+                    if ("item".equals(groupNode.getNodeName())) {
+                        final List<String> checkNamesOrder = getCurrentNamesOrder(groupNode);
+                        final List<String> sortedCheckNames = new ArrayList<>(checkNamesOrder);
+                        Collections.sort(sortedCheckNames);
+                        assertWithMessage("Check Names must be in alphabetical Order.")
+                                .that(checkNamesOrder)
+                                .containsExactlyElementsIn(sortedCheckNames)
+                                .inOrder();
+                        index++;
+                    }
+                    groupNode = groupNode.getNextSibling();
+                }
+            }
+            if ("Filters".equals(XmlUtil.getNameAttributeOfNode(current))) {
```

Comment:
please add FileFilters, even it has only 1 for now, it will grow in future

Reply:
done